x-pack/plugins/security_solution/public/common/components/links/index.tsx

```diff
@@ -56,7 +56,8 @@ const UserDetailsLinkComponent: React.FC<{
   title?: string;
   isButton?: boolean;
   onClick?: (e: SyntheticEvent) => void;
-}> = ({ children, Component, userName, isButton, onClick, title }) => {
+  testingId?: string;
+}> = ({ children, Component, userName, isButton, onClick, title, testingId }) => {
```

Comment:
We've been using `dataTestSubj` in the file, I think it'd be better to align with the existing naming.

Reply:
I opted to not modify this file in favour of adding "data-test-subj" to the column cells instead:

```
 {
    field: 'userName',
    name: i18n.USER_ALERTS_USERNAME_COLUMN,
    truncateText: true,
    textOnly: true,
    'data-test-subj': 'userSeverityAlertsTable-userName',  <---
    render: (userName: string) => <UserDetailsLink userName={userName} />,
  },
  ```